journals/apps/journals/templatetags/wagtail_tags.py

```diff
@@ -20,3 +22,8 @@ def collections_user_has_access(collections, request):
 
     """
     return WagtailModelManager.get_user_collections(request.user, collections)
+
+
+@register.filter(name='md5hash')
```

Comment:
I think we should change this filter name to something like `block_anchor_id`

Reply:
yeah, sounds good